mkdocs/docs/api.md

```diff
@@ -273,6 +273,10 @@ tbl.append(df)
 
 # or
 
+tbl.merge_append(df)
```

Comment:
Sounds great! I am also +1 on let it controlled by the config. I made `merge_append` a separate API to mirror the Java side implementation, which has `newAppend` and `newFastAppend` APIs. But it seems better to just make the `commit.manifest-merge.enabled` default to `False` on python side.

I will still keep FastAppend and MergeAppend as separate class, and keep `merge_append` in `UpdateSnapshot` class to ensure clarity, although the current MergeAppend is purely FastAppend + manifest merge.

Just curious, why not Java side `newAppend` return an `FastAppend` impl when `commit.manifest-merge.enabled` is False. Is it due to some backward compatibiilty issue?

Reply:
Thanks! I think the use-case of the Java library is slightly different, since that's mostly used in query engines.

> Is it due to some backward compatibiilty issue?

I think it is for historical reasons, since the fast-append was added later on :)

Reply:
btw, I like how you split it out in classes, it is much cleaner now 👍